addons/mail/static/src/js/filter_menu.js

```diff
@@ -7,6 +7,8 @@ odoo.define('mail.FilterGeneratorMenu', function (require) {
     utils.patch(FilterGeneratorMenu, 'mail.FilterGeneratorMenu', {
 
         /**
+         * With the `mail` module installed, we want to filter out some of the
+         * available fields.
```

Comment:
in 'Add Custom Filter' menu (FilterGeneratorMenu)?

Reply:
Ok